### ISSUE TITLE
Empty trans-units since 2.0.0

### ISSUE THREAD

**Issue by laverix** [jaroslavsvak/xlf-merge#10]
Hello, thanks for your effort! Great library!

I figured out on my project that since version "2.0.0" it generates empty file with all `<trans-unit></trans-unit>` in it.

**Comment by finnporter** [comment]
Yeah I'm having the same issue. it's gives me a success message and I end up with a minified xml file that has a whole lot of trans units and ids but no actual translated text. Is there any way to fix that?
